cloud: document Azure Blob Storage Private Link import (#22427)
pingcap/docs: tidb-cloud/import-csv-files.md

    - **Included Schema Files**: if the source folder contains the target table schema files (such as `${db_name}-schema-create.sql`), select **Yes**. Otherwise, select **No**.
    - **Data Format**: select **CSV**.
    - **Connectivity Method**: select how TiDB Cloud connects to your Azure Blob Storage:

        - **Public** (default): connects over the public internet. Use this option when the storage account allows public network access.
        - **Private Link**: connects through an Azure private endpoint for network-isolated access. Use this option when the storage account blocks public access or when your security policy requires private connectivity. If you select **Private Link**, you need to provide the following additional field **Azure Blob Storage Resource ID**.  To find the resource ID:
            
            1. Go to the [Azure portal](https://portal.azure.com/).

>             2. Navigate to your storage account, click **Overview** > **JSON View**.

Reviewer: ```suggestion
            2. Navigate to your storage account and click **Overview** > **JSON View**.
```

>         - **Azure Blob Storage Resource ID**: enter the resource ID of your Azure Storage account. To find the resource ID, go to the [Azure portal](https://portal.azure.com/), navigate to your storage account, click **Overview** > **JSON View**, and copy the value of the `id` property. The resource ID is in the format `/subscriptions/<subscription_id>/resourceGroups/<resource_group>/providers/Microsoft.Storage/storageAccounts/<account_name>`.

Reviewer 2: ```suggestion
        - **Public** (default): connects over the public internet. Use this option when the storage account allows public network access.
        - **Private Link**: connects through an Azure private endpoint for network-isolated access. Use this option when the storage account blocks public access or when your security policy requires private connectivity. If you select **Private Link**, you need to provide the following additional field **Azure Blob Storage Resource ID**.  To find the resource ID:
            
            1. Go to the [Azure portal](https://portal.azure.com/).
            2. Navigate to your storage account, click **Overview** > **JSON View**.
            3. Copy the value of the `id` property. The resource ID is in the format `/subscriptions/<subscription_id>/resourceGroups/<resource_group>/providers/Microsoft.Storage/storageAccounts/<account_name>`.
```

    - **SAS Token**: enter an account SAS token to allow TiDB Cloud to access the source files in your Azure Blob Storage container. If you don't have one yet, you can create it using the provided Azure ARM template by clicking **Click here to create a new one with Azure ARM template** and following the instructions on the screen. Alternatively, you can manually create an account SAS token. For more information, see [Configure Azure Blob Storage access](/tidb-cloud/dedicated-external-storage.md#configure-azure-blob-storage-access).

4. Click **Connect**.

    If you selected **Private Link** as the connectivity method, TiDB Cloud creates a private endpoint for your storage account. You need to approve this endpoint request in the Azure portal before the connection can proceed:

    1. Go to the [Azure portal](https://portal.azure.com/) and navigate to your storage account.
    2. Click **Networking** > **Private endpoint connections**.
    3. Find the pending connection request from TiDB Cloud and click **Approve**.

>     4. Return to TiDB Cloud. The import wizard proceeds automatically once the endpoint is approved.

Reviewer 2: ```suggestion
    4. Return to the [TiDB Cloud console](https://tidbcloud.com/). The import wizard proceeds automatically once the endpoint is approved.
```

    > **Note:**
    >

>     > If the endpoint is not yet approved, TiDB Cloud displays the following message: "Connection pending. Please approve the Private Endpoint request in your Azure Portal settings and try again." Approve the request in the Azure portal and retry the connection.

Reviewer 2: ```suggestion
    > If the endpoint is not yet approved, TiDB Cloud displays the following message: "Connection pending. Please approve the Private Endpoint request in your Azure Portal settings and try again." Approve the request in the [Azure portal](https://portal.azure.com/) and retry the connection.
```
